testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/DeleteAccountActionConfirmPage.java

```diff
@@ -0,0 +1,40 @@
+package org.keycloak.testsuite.ui.account2.page;
+
+import org.keycloak.testsuite.auth.page.AuthRealm;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.keycloak.testsuite.util.UIUtils.clickLink;
+
+public class DeleteAccountActionConfirmPage extends AuthRealm {
```

Comment:
This should extend the [`RequiredActions`](https://github.com/keycloak/keycloak/blob/65f51b7b83271bc42de923b3625cac8a69e8e9a2/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/auth/page/login/RequiredActions.java) class, similarily to e.g. [`UpdatePassword`](https://github.com/keycloak/keycloak/blob/65f51b7b83271bc42de923b3625cac8a69e8e9a2/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/auth/page/login/UpdatePassword.java) page object. Should be also placed in the same package to be available for other tests too.

Reply:
:+1: